### DISCUSSION TITLE
Add an about:blank cloaker on the home page

### DISCUSSION THREAD

**Discussion by okimcool-0** [UseInterstellar/Interstellar#693] [Ideas]
Somehow my school managed to automatically block everything except for the home page on every link except github codespace links and so when I try using one, the home page isn't blocked, but games, apps, tabs, and settings is. However, Doge Unblocker has
 a way to use about:blank without leaving the homepage and that fixes it getting automatically blocked, but this doesn't exist for interstellar (My school also blocked all popups and redirects so automatic cloaking doesn't work.) 
[Screen recording 2024-05-09 9.54.44 AM.webm](https://github.com/UseInterstellar/Interstellar/assets/109975818/18fd67c1-4fc2-41a5-a08d-183a501c35f7)

**Comment by replitidk** [comment]
its a ok idea but if you want it added put it in issues with a request label on it

**Comment by replitidk** [comment]
also your school likely got smart and blocked the /ap for apps and the /g for games

**Comment by nvd123YT** [comment]
ACTUALLY SMART SCHOOL?!?!?!?!?!??!?!?!?!? lol

**Comment by 0mhx** [comment]
the paths got changed. are you still having this issue?

This issue will be closed in 1 week if there's no response

**Comment by 0mhx** [comment]
inactive
